src/Reducer/cartReducer.jsx

```diff
@@ -1,22 +1,22 @@
 export function cartReducer(state, action) {
-    switch(action.type) {
 
+    switch(action.type){
+        case "GET_CARTITEM_FROM_LOCAL_STORAGE":
```

Comment:
Suggestion: You could destructure action object for ease of usability

Example: { type, payload}